turbopack/crates/turbopack-tests/tests/execution/turbopack/exports/invalid-export/input/index.js

```diff
@@ -1,3 +1,10 @@
 it('should error when importing an invalid export', () => {
-  expect(require('./invalid-export').default).toBe(undefined)
+  // Either requiring should throw, or return undefined
+  let ns
+  try {
+    ns = require('./invalid-export')
```

Comment:
Wouldn't this pass when export fails?

Reply:
There is a Turbopack build error issue being emitted, which fails the build anyway.
And I would say Turbopack should just create any code here, it doesn't matter what because the main thing will be the build error anyway.